<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="PlotWidget" name="canvas1"/>
    </item>
    <item>
     <widget class="PlotWidget" name="canvas2"/>
    </item>
    <item>
     <widget class="PlotWidget" name="canvas3"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="xmin">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>170</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xmax">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>190</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="dr_slider">
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="dr">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSlider" name="ds_slider">
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ds">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dr_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dr</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dr_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>617</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ds_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ds</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>644</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ds</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ds_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
